bin/init.sh

```diff
@@ -48,17 +48,17 @@ export GOARCH=${GOARCH:-'amd64'}
 LOCAL_OS="`uname`"
 case $LOCAL_OS in
   'Linux')
-    OS='Linux'
+    LOCAL_OS='linux'
```

Comment:
are you sure OS isn't used anywhere ?
also this section now seems to just be `tolower(uname)`

Reply:
I just did what the Makefile is doing - someone broke running init.sh ( and localTestEnv.sh ) if not called from make.

Reply:
Well - a lot was broken in init.sh, no clue who reviewed and made the change, or why - but the 
scripts have been usable outside of make and should continue to be usable.

Reply:
I was more concerned about what might broke with this change

I think I remember a PR trying to make it fail fast if not invoked from the Makefile to avoid a bunch of duplicate logic, though not sure that one got merged or not

Why should it work standalone / which use case is it ?

Reply:
Why break it intentionally ? I am using it, I assume others are using the scripts as well,
and normal expectation for a shell script is to work when called - no need to justify use 
cases for it. 
It doesn't make sense to have scripts that can only be called from Makefile.